docs/product/explore/session-replay/mobile/performance-overhead.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "Replay Performance Overhead"
+sidebar_order: 30
+description: "Learn more about how enabling Session Replay impacts the performance of your application."
+---
+
+Session Replay for mobile works by taking snapshots of the view hierarchy as well as a screenshot within the same frame, at the rate of one per second. These screenshots are then compressed into a video file representing a segment of the user’s session. All these small segments are then streamed into Sentry with additional trace identifiers, breadcrumbs, and other useful debugging information to compose the full user session.
+
+In order to perform this work without negatively impacting the host application's performance, the mobile SDKs take care to introduce minimal additional CPU and memory overhead, record screenshots and view hierarchy in a non-intrusive way and only when necessary, send the absolute minimum number of bytes required, and use low-latency ingestion endpoints geographically close to your end-users.
+
+**For most mobile applications, the performance overhead of our mobile SDKs will be imperceptible to end-users.**
+
+## How Is Session Replay Optimized?
+
+Sentry's Session Replay SDK takes several measures to avoid negatively impacting the performance of the screen on which it's running:
```

Comment:
We say "Mobile SDKs" above... which makes more sense, "Mobile SDK" or "Session Replay SDK"?

We should try to be consistent.

Also is there a reason why we specify "screen" instead of "of your application"?

Reply:
I think Session Replay SDK makes more sense. As in, "the session replay part of the mobile sdk"